core/storage/base_model/gae_models.py

```diff
@@ -1184,6 +1188,13 @@ class BaseSnapshotContentModel(BaseModel):
     # The snapshot content, as a JSON blob.
     content = ndb.JsonProperty(indexed=False)
 
+    @staticmethod
+    def get_export_policy():
+        """The contents of snapshots are not relevant to the user for
+        Takeout.
+        """
+        return EXPORT_POLICY.NOT_APPLICABLE
+
```

Comment:
Why is this not applicable, shouldn't we return the content when the user is author of this snapshot? (maybe not I don't exactly remember what we decided to do)

Reply:
We concluded that the content of the snapshot wasn't relevant to the user for the purposes of Takeout.